src/scripts.js

```diff
@@ -29,14 +32,14 @@ const upcomingVisitsContainer = document.getElementById('upcomingVisitsContainer
 const pastVisitsCardsContainer = document.getElementById('pastVisitsCardsContainer');
 const availableRoomsCardsContainer = document.getElementById('availableRoomsCardsContainer')
 const currentVisitsCardsContainer = document.getElementById('upcomingVisitsCardsContainer');
+const apologeticMessageContainer = document.querySelector('.apologetic-message-container');
```

Comment:
You might think about moving some of these query selectors into the domUpdates file instead of passing them as arguments. It might make it a bit cleaner and get some of the query selectors out of your scripts file